src/components/CardContainer/CardContainer.js

```diff
@@ -7,7 +7,8 @@ class CardContainer extends React.Component {
         super(props)
 
         this.state = {
-            dogs: []
+            dogs: [],
+            fetchUrl: 'http://localhost:3000/dogs'
```

Comment:
Could be declared as a const in the component rather than state so that I can't be changed by the component.